### DISCUSSION TITLE
Plot discrete colormap

### DISCUSSION THREAD

**Discussion by YosmelyBermudez** [pydata/xarray#5952] [General]
Hello! I need to plot a seismic horizon, and the variables are inline, xline, time and preditions, where predictions are values ​​from 0 to 3 and they will be specified in a colorbar. Attached image. How can I specify the axes? How can I plot this image? Notice that the colors of the colorbar are not in the image yet, it is just what I have in mind.

![what i want](https://user-images.githubusercontent.com/76023258/140756129-651b27f2-58ef-491a-a88b-c4ada4da37f3.jpg)

**Comment by ahuang11** [comment]
https://stackoverflow.com/questions/53360879/create-a-discrete-colorbar-in-matplotlib

or 
http://holoviews.org/user_guide/Styling_Plots.html
